src/gennodejs/generate.py

```diff
@@ -230,9 +230,9 @@ def write_requires(s, spec, previous_packages=None, prev_deps=None, isSrv=False)
     if previous_packages is None:
         s.write('"use strict";')
         s.newline()
-        s.write('let _serializer = require(\'../../../base_serialize.js\');')
-        s.write('let _deserializer = require(\'../../../base_deserialize.js\');')
-        s.write('let _finder = require(\'../../../find.js\');')
+        s.write('let _serializer = require(\'../base_serialize.js\');')
+        s.write('let _deserializer = require(\'../base_deserialize.js\');')
+        s.write('let _finder = require(\'../find.js\');')
```

Comment:
Ah. This answers my first question. Still, I don't think the `CMakeLists.txt` copies are necessary.